docs/src/main/sphinx/security/inspect-jks.rst

```diff
@@ -0,0 +1,148 @@
+.. _jks_certs:
+
+=========
+JKS files
+=========
+
+This topic describes the :ref:`Java KeyStore (JKS) <glossJKS>` certificate
+container formats accepted by Trino to validate TLS/HTTPS client access to the
+cluster.
+
+.. _server_java_keystore:
+
+Java keystore
+-------------
+
+.. note::
+
+  See :ref:`Inspect PEM files <pem_certs>` for information on the preferred
+  certificate format.
+
+The Java KeyStore (JKS) system is provided as part of your Java installation.
+Private keys and certificates for your server are stored in a *keystore* file.
+The keystore file is always password-protected.
+
+To be recognized as valid by the Trino :doc:`CLI </installation/cli>` and modern
+browsers, the key must be signed by a trusted Certificate Authority (CA). For
+JKS keys, this means either that a certificate for the key's signing authority
+is listed in the current JVM's :ref:`truststore <cli_java_truststore>`, or that
+the certificate received from a CA is chained all the way back to a CA in the
+truststore.
```

Comment:
I would drop the "or" clause.  I find it is confusing extra details

Reply:
done